Lib/tkinter/__init__.py

```diff
@@ -280,13 +280,24 @@ def NoDefaultRoot():
     Call this function to inhibit that the first instance of
     Tk is used for windows without an explicit parent window.
     """
-    global _support_default_root
-    _support_default_root = 0
-    global _default_root
+    global _support_default_root, _default_root
+    _support_default_root = False
     _default_root = None
```

Comment:
Why rebind before deleting?

Reply:
Repeated call should not fail.

Added comment and tests for this.

Reply:
Ah.  Thank you.